app/pages/notifications/notification-section.jsx

```diff
@@ -140,6 +139,19 @@ export default class NotificationSection extends Component {
     });
   }
 
+  markAsRead(readNotification) {
+    const { notificationsCounter } = this.context;
+    const { user } = this.props;
+    const { notifications } = this.state;
+    const relatedNotifications = notifications.filter(
```

Comment:
I think i'm just missing something but why are we updating the related notifications here? 

How is the notification i just read linked to another and why do i want to mark that as 'read' as well?

Reply:
Is this the case where I click on one message in a discussion so I've read all the preceding new messages too, without having to go back up the page and individually click each one.

I often get a series of notifications for the same discussion, but only click through to one of them in order to read them all.

Reply:
Good question and I'm not tied to current code, but intention is if you have 3 notifications related to the same discussion, clock notification to go to that discussion, it'd mark all the related notifications as read as they'd have likely been shown in the discussion page you're now on.